### PR TITLE
Static Page

### PR DESCRIPTION
**Pull request by NgengChhengkim**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
+  root "staticpage#home"
+  get "home"=>"staticpage#home"
```
Comment (yongsokheng) [inline config/routes.rb]: No need get "home"=>"staticpage#home"

File: app/views/staticpage/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<% provide :title, "Home" %>
+<div class="center btn-flat disabled button">
+  <h2><%= t("labels.welcome") %></h2>
```
Comment (yongsokheng) [inline app/views/staticpage/home.html.erb]: Delete ()

File: Gemfile [hunk 3]
```diff
@@ -1,6 +1,10 @@
-source "https://rubygems.org"
-# Bundle edge Rails instead: gem "rails", github: "rails/rails"
-gem "rails", "4.2.1"
+source 'https://rubygems.org'
+
```
Comment (RathanakSreang) [inline Gemfile]: use `"` instead of `'`

File: Gemfile [hunk 4]
```diff
@@ -1,6 +1,10 @@
...
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '4.2.1'
```
Comment (RathanakSreang) [inline Gemfile]: use `"` instead of `'`

File: Gemfile [hunk 5]
```diff
@@ -29,6 +33,10 @@ gem "sdoc", "~> 0.4.0", group: :doc
 # gem "capistrano-rails", group: :development
 gem "mysql2"
 gem "devise"
+# gem 'capistrano-rails', group: :development
+# Use mysql as the database for Active Record
+gem 'materialize-sass'
```
Comment (RathanakSreang) [inline Gemfile]: use `"` instead of `'`

**Comment by RathanakSreang** [comment]
Ok

**Pull request merged**